### DISCUSSION TITLE
How to link a external URL to a Post in Listings Section.

### DISCUSSION THREAD

**Discussion by ANP-Oxy** [quarto-dev/quarto-cli#6338] [Q&A]
### Description

Hello quarto community and Thankyou in advance for the help. I am trying to create a simple portfolio website for myself with quarto in which I have  a landing page which has bio about me and a projects page which lists various data analysis projects I have completed. below you can see the preview. 
![image](https://github.com/quarto-dev/quarto-cli/assets/94713251/ef58a5a3-d63b-46df-bf34-f07e60408cab)
Now when I click on the projects it looks like this 
![image](https://github.com/quarto-dev/quarto-cli/assets/94713251/e80f820a-cbf1-464f-8678-af94a294fb4d)
Now problem is I have uploaded projects to github already so I want to link those urls to the posts shown in projects page so when someone clicks on specific  project it should take it to that page on my github account. 

I tried doing that by creating a listing using a .yml file but it didn't work. for some reason it does not redirect to the external url provided in the "href" argument 
![image](https://github.com/quarto-dev/quarto-cli/assets/94713251/c747d2ed-8e92-4e7a-8725-3c7e3bcfc864)

This is how my projects page looks 
![image](https://github.com/quarto-dev/quarto-cli/assets/94713251/b19e21b0-e189-420f-9766-ed383db71aa4)

so I just want to know if there is way to do this instead of creating a whole .qmd file for the project again and linking that to the listing.

**Comment by mcanouil** [comment]
Duplicate of / See <https://github.com/quarto-dev/quarto-cli/discussions/6309>.

**Reply by ANP-Oxy** [reply]
ok so I tried this but it is giving me this OS error 
![image](https://github.com/quarto-dev/quarto-cli/assets/94713251/334c6dd6-4479-4143-8995-2f52aac7e2cb)
Not sure what exactly this means

**Reply by dragonstyle** [reply]
What version of Quarto are you using? Perhaps it is an older version?

**Reply by ANP-Oxy** [reply]
I have the latest version installed.  It seems like this problem is due to windows and not quarto but I don't really know how to solve it. For now I'm just using html redirect to link it to the external URLs

**Reply by dragonstyle** [reply]
I was able to confirm a bug specific to Windows which I just fixed. Our next pre-release build will have the fix. Thanks for letting us know about this issue!

**Reply by SimonCoulombe** [reply]
Hi ! I  get this issue on a clean install of windows 11 and quarto 1.4  from CRAN   Which build has the fix?  
thanks

**Reply by mcanouil** [reply]
The R package is not Quarto CLI.
It is only a means to use Quarto CLI from R.

From a terminal (not R) use:
```sh
quarto check
```

This will check your setup and display the version of Quarto CLI (unrelated to the version of the R package).

See <https://quarto.org/docs/get-started/>.

**Reply by SimonCoulombe** [reply]
you're right, the quarto  version packaged with rstudio was 1.3x.   downloaded quarto, installed it, rstarted rstudio and now it works.

![image](https://github.com/quarto-dev/quarto-cli/assets/23444900/d950543d-3e48-4351-933f-4c61961ea0d3)
